cmd/influxd/inspect/dump_tsm/dump_tsm_test.go

```diff
@@ -201,6 +201,7 @@ func makeTSMFile(t *testing.T, params tsmParams) (string, string) {
 	for _, key := range params.keys {
 		values := []tsm1.Value{tsm1.NewValue(0, 1.0)}
 		require.NoError(t, w.Write([]byte(key), values))
+		require.NoError(t, w.Flush())
 	}
 	if len(params.keys) != 0 {
 		require.NoError(t, w.WriteIndex())
```

Comment:
Can't comment on it directly, but do we also need to `Flush()` after [this line](https://github.com/influxdata/influxdb/pull/22134)? The failures I've seen in CI all mention `foobar` so I'd expect the fix to have something to do with that.

Reply:
Flush will be called by the `w.Close()` below so that should be okay